android/src/main/java/io/customer/reactnative/sdk/messaginginapp/ReactInlineInAppMessageView.kt

```diff
@@ -19,10 +21,24 @@ class ReactInlineInAppMessageView @JvmOverloads constructor(
     @StyleRes defStyleRes: Int = 0
 ) : WrapperInlineView<ReactInAppPlatformDelegate>(
     context, attrs, defStyleAttr, defStyleRes
-) {
+), InlineMessageActionListener {
     override val platformDelegate = ReactInAppPlatformDelegate(view = this)
 
     init {
         initializeView()
+        setActionListener(this)
+    }
+
+    override fun onActionClick(message: InAppMessage, actionValue: String, actionName: String) {
+        val payload = mapOf(
+            "message" to mapOf(
+                "messageId" to message.messageId,
+                "deliveryId" to message.deliveryId,
+                "elementId" to elementId
+            ),
+            "actionValue" to actionValue,
+            "actionName" to actionName
+        )
+        platformDelegate.dispatchEventInternal(ReactInAppPlatformDelegate.ON_ACTION_CLICK, payload)
```

Comment:
if the value above is null? wouldn't that cause exception 
```
payload = Arguments.makeNativeMap(payload),
```

Reply:
I believe React Native allows passing `null` values in `Arguments.makeNativeMap()`. I also did a quick test by forcing `null` values and it worked fine, so think we're good here.